client/mass/test/cuminc.integration.spec.js

```diff
@@ -300,23 +307,41 @@ tape('term1 = Cardiovascular System, term2 = agedx, numeric regular bins', test
 			`Should correctly pass q.first_bin.stop = ${testStop} to config`
 		)
 
-		//Test q.bin_size and q.first_bin.stop changes are applied
-		const newStop = 1
-		cuminc.Inner.config.term2.q.bin_size = 3
-		cuminc.Inner.config.term2.q.first_bin.stop = newStop
-
-		await cuminc.Inner.app.dispatch({
-			type: 'plot_edit',
-			id: cuminc.Inner.id,
-			config: cuminc.Inner.config
+		//***Test q.bin_size and q.first_bin.stop changes are applied
```

Comment:
@gavrielm I changed from looking for pill changes to changes in the plot, like this example here. Please let me know if you agree.